routers/web/repo/issue.go

```diff
@@ -2194,7 +2194,8 @@ func UpdateCommentContent(ctx *context.Context) {
 	if !ctx.IsSigned || (ctx.User.ID != comment.PosterID && !ctx.Repo.CanWriteIssuesOrPulls(comment.Issue.IsPull)) {
 		ctx.Error(http.StatusForbidden)
 		return
-	} else if comment.Type != models.CommentTypeComment && comment.Type != models.CommentTypeCode {
+	}
+	if comment.Type != models.CommentTypeComment && comment.Type != models.CommentTypeReview {
```

Comment:
What happened to the `comment.Type != models.CommentTypeCode`?
Won't that break something else?
Or was that not supposed to be present?

Reply:
Fixed